CONTRIBUTING.adoc

```diff
@@ -156,17 +188,30 @@ NOTE: This is a notice that you should pay attention to!
 CAUTION: This is a common mistake!
 
 
-There are too many other helpful macros and formatting options to list here, so it
-is recommended that you refer to the link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference[quick reference]
+There are too many other helpful macros and formatting options to list here, so
+it is recommended that you refer to the
+link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference[quick reference]
 to become more familiar with what is available.
 
 
+==== Adding a Tutorial page
+
+A tutorial is presented on its own page, each of which is written up as an
+`.adoc` file located in the link:content/doc/tutorials[`content/doc/tutorials/`]
```

Comment:
Should this link point to the a github url:
Example:
```
link:https://github.com/jenkins-infra/jenkins.io/tree/master/content/doc/tutorials[`content/doc/tutorials/`]
```

Reply:
These should all resolve to Github URLs, but did you mean pointing these types of links to the relevant directories within the jenkins-infra/jenkins.io repo (as full absolute URLs)?

Actually, that's what I initially thought about doing before my first commit on this pull request. However, I changed my mind (and made them relative rather than absolute links) because I wanted to direct the reader to the relevant directories/files in *their own fork* of jenkins.io. I suspect if a reader were to view this page within the upstream jenkins-infra/jenkins.io repo, they would get to the relevant directory/file in  jenkins-infra/jenkins.io repo. I've checked that these relative links work in my own branch (and I would hope they would still work if this pull request were merged).

Having said that, I'm also happy to change these to full absolutely URLs (within the jenkins-infra/jenkins.io repo).

Reply:
@gilesgas - You are correct.  Thanks for explaining.